src/Containers/Microsoft.NET.Build.Containers/Tasks/ComputeDotnetBaseImageAndTag.cs

```diff
@@ -278,4 +296,82 @@ private bool ComputeRepositoryAndTag([NotNullWhen(true)] out string? repository,
                 return null;
         };
     }
+
+    private bool UserImageIsMicrosoftBaseImage => UserBaseImage?.StartsWith("mcr.microsoft.com/dotnet") ?? false;
+
+    private void LogNoInferencePerformedTelemetry()
+    {
+        // we should only log the base image, tag, containerFamily if we _know_ they are .NET's MCR images
+        string? userBaseImage = null;
+        string? userTag = null;
+        string? containerFamily = null;
+        if (UserBaseImage is not null && UserImageIsMicrosoftBaseImage)
+        {
+            if (ContainerHelpers.TryParseFullyQualifiedContainerName(UserBaseImage, out var containerRegistry, out var containerName, out var containerTag, out var _, out bool isRegistrySpecified))
+            {
+                userBaseImage = $"{containerRegistry}/{containerName}";
```

Comment:
Is Container Name a unique user input name? If so, you may want to store the registry in its own variable, considering this data will get tossed once it gets through the GDPR layer.

Reply:
In this case it should _never_ be user input - we are checking that the image comes from `mcr.microsoft.com` before doing this, so we should have a very constrained set of values - these are all images that MS owns and pushes, never user-facing images.